src/main/java/racingcar/vo/CarList.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.vo;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class CarList {
+    private static final String STRING_WINNER_DELIMITER = ", ";
+    private final List<Car> carList;
+
+    public CarList(final List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public List<Car> getCarList() {
+        return carList.stream()
+                .collect(Collectors.toUnmodifiableList());
+    }
+
+    public void moveForward() {
+        carList.stream()
+                .forEach(Car::moveForward);
+    }
+
+    public List<Car> getCarListWithLongestDistance() {
+        final int longestDistance = getLongestDistanceFromCarList();
+        final List<Car> carListWithLongestDistance = filterCarListWithLongestDistance(longestDistance);
+        return Collections.unmodifiableList(carListWithLongestDistance);
+    }
+
+    private int getLongestDistanceFromCarList() {
+        final int longestDistance = carList.stream().mapToInt(Car::getDistance).max().orElse(0);
+        return longestDistance;
+    }
+
+    private List<Car> filterCarListWithLongestDistance(final int longestDistance) {
+        return carList.stream()
+                .filter(car -> car.getDistance() == longestDistance)
+                .collect(Collectors.toUnmodifiableList());
+    }
+
+    public String getWinners() {
+        final List<Car> carListWithLongestDistance = getCarListWithLongestDistance();
+        final List<String> winnerList = carListWithLongestDistance.stream()
+                .map(Car::getName).collect(Collectors.toUnmodifiableList());
```

Comment:
```suggestion
        final List<String> winnerList = carListWithLongestDistance.stream()
                .map(Car::getName)
                .collect(Collectors.toUnmodifiableList());
```
스트림의 가독성을 위해 이렇게 개행을 하면 좋을 것 같아요!

Reply:
@skylar1220 감사합니다! 인텔리제이에서 자동으로 포맷팅되는 기능을 썼는데, 개행 없이 만들어주네요. 😭😭 그 기능을 꺼야할 까 봐요.